source/examples/generated/android/local/AsyncTest.codeblock.realm-async-task.kt

```diff
@@ -0,0 +1,18 @@
+// using class instances for transaction, success, error
+realm.executeTransactionAsync(Realm.Transaction { transactionRealm ->
+        val item: Item = transactionRealm.createObject(Item::class.java)
```

Comment:
```suggestion
        val item: Item = transactionRealm.createObject<Item>()
```

There is a Kotlin extension function for this.